app/data/lar-sphere.gdml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
One thing to keep in mind if we add a copy of the gdml to the app directory is we may need to update both versions as we add more optical processes/properties.